app/web/features/dashboard/locales/en.json

```diff
@@ -1,33 +1,35 @@
+  "outreach_title": "Grow the platform",
+  "outreach_pill": "Outreach",
+  "outreach_subtitle": "Your help is needed to spread the word",
+  "outreach_markdown": "We cannot build this platform without your help. If you want to see Couchers.org grow, there are many ways you can help.\n\n**Tell your friends**: If anyone you know would be excited about this new platform, please tell them! If they're active couch surfers and you trust them, we want them here!\n\n**Social media**: Spread the word on your social media platforms. If you are part of any groups or discussions (e.g. Facebook or WhatsApp groups), please post there. Even better, try to talk to moderators and convert existing groups to include Couchers.org, like they did in the [Chicago community](https://www.facebook.com/groups/cschicago/).\n\n**Talk about us at events**: If you head to any events listed on other couch surfing platforms, mention us to people there!\n\n**YouTubers, Bloggers, Influencers and Media**: If you know any people out there making videos, blogs, or posts about couch surfing or travel in general, talk to them and see if they would like to make any content about the platform. Get in contact with local news outlets or magazines and see if they would be interested in hearing about Couchers.org.",
+  "contribute_title": "Sign up to contribute",
+  "contribute_pill": "Volunteer",
+  "landing_text": "We are building new features like events, local guides, moderation and hangouts. We appreciate your patience and support as we develop these.",
+  "your_communities_helper_text": "You have been added to all communities based on your location. Feel free to browse communities in other locations as well.",
+  "your_communities_helper_text2": "Don't see your community? Then it's waiting for you to get it started!",
+  "community_builder_form_text": "Start your own local community",
+  "community_builder_form_link": "https://couchers.org/community-builder-form",
```

Comment:
This should stay in the constant file since it doesn't need to be translated

Reply:
Fixed with ab59146a977ab3f50aff9603a87778c7fd4f2e9b